packages/cozy-client/src/node.js

```diff
@@ -31,3 +31,5 @@ export * from './cli'
 
 import * as models from './models'
 export { models }
+
+export * from './hooks'
```

Comment:
Why do we want to export hooks for the node version?

Reply:
Currently, jest is based on our `main` entry point aka our node entry and not our browser one.

Reply:
Mmh ok, but that means `react` is imported on node again, and I think that was a problem?

Reply:
fixed https://github.com/cozy/cozy-client/pull/651/commits/1b049d71996fe7ef7f065bf433c7ac649a824ef7